src/e2e-test/features/bigqueryexecute/BQExecute.feature

```diff
@@ -124,7 +124,7 @@ Feature: BigQueryExecute - Verify data transfer using BigQuery Execute plugin
     Then Verify the pipeline status is "Succeeded"
     Then Verify 1 records inserted in BigQuery table: "bqSourceTable" with query "bqExecuteCountDMLInsert"
 
-  @BQ_SOURCE_TEST @BQ_EXECUTE_UPSERT_SQL @BQExecute_Required
```

Comment:
why removing this test?

Reply:
No specific reason, Added this one ```Verify BQExecute plugin functionality for DDL query - Create table``` to required list so to balance the build time removed this one. (This feature file already have 2 tests marked as required)

Reply:
Do you have the data how much time `bq execute` required tests takes to execute in comparison to big ones like `gcs` and `bigquery`? If it is smaller then I think it should be fine to add it back because the overall execution time will remain the same.